test/testcafe/framework/page-objects/components/CalloutObject.js

```diff
@@ -13,22 +15,24 @@ export default class CalloutObject {
   }
 
   isWarning() {
+    if (userVariables.gen3) {
+      return this.el.hasClass('MuiAlert-calloutWarning');
+    }
     return this.el.hasClass('infobox-warning') &&
       this.el.child('[data-se="icon"]').hasClass('warning-16');
   }
 
   isError() {
-    const hasInfoBoxErrorClass = this.el.hasClass('infobox-error');
     if (userVariables.gen3) {
-      return hasInfoBoxErrorClass;
+      return this.el.hasClass('MuiAlert-calloutError');
```

Comment:
Assuming my comment about `[data-se="message"]` not being used in our page objects is true, we can switch these selectors to target `[data-se="infobox-error"]`, `[data-se="infobox-warning"]`, etc.

Reply:
Done
https://github.com/okta/okta-signin-widget/pull/3484/commits/11c32491a5bdbde59e9f375e6bda4c5fd31587f5#diff-4bcc86ed383d602f2dbea0af4ee097d300d074ec0c3b90f1319dedd8266197a9R25